docs/reST/ref/pygame.rst

```diff
@@ -192,6 +192,25 @@ object instead of the module, which can be used to test for availability.
 
    .. ## pygame.encode_file_path ##
 
+.. function:: debug
```

Comment:
does this documentation need updating to match the renamed function?

Reply:
Just pushed a commit to fix this